PayKids/presentation/src/main/java/com/paykids/presentation/view/diary/CategoryConsumeAdapter.kt

```
@@ -0,0 +1,70 @@
+            itemView.setOnClickListener {
+
+            }
+        }
+
+        private fun formatToMonthDay(date: String): String {
+            return try {
+                val originalFormat = SimpleDateFormat("yyyy-MM-dd", Locale.getDefault())
+                val targetFormat = SimpleDateFormat("MM/dd", Locale.getDefault())
+                val parsedDate = originalFormat.parse(date)
+                targetFormat.format(parsedDate ?: "")
+            } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)